mp/src/public/tier1/iconvar.h

```diff
@@ -67,7 +67,6 @@ class CCommand;
 
 #define FCVAR_NOT_CONNECTED		(1<<22)	// cvar cannot be changed by a client that is connected to a server
 #define FCVAR_MATERIAL_SYSTEM_THREAD (1<<23)	// Indicates this cvar is read from the material system thread
-#define FCVAR_ARCHIVE_XBOX		(1<<24) // cvar written to config.cfg on the Xbox
```

Comment:
I would add this to the comments below with `// #define FCVAR_AVAILABLE (1<<24)`

Reply:
Somehow didn't notice those comments, Will do.